src/components/EntityPage/Participant/ParticipantClinical.js

```diff
@@ -147,16 +147,16 @@ class ParticipantClinical extends React.Component {
 
     const diagHeads = [
       { Header: 'Diagnosis Category', accessor: 'diagnosis_category', Cell: cellBreak },
-      { Header: 'Diagnosis (Mondo)', accessor: 'mondo_id_diagnosis', Cell: cellBreak },
-      { Header: 'Diagnosis (NCIT)', accessor: 'ncit_id_diagnosis', Cell: cellBreak },
+      { Header: 'Diagnosis (Mondo)', accessor: 'mondo_id_diagnosis', Cell: (wrapper) => wrapper.value === "--" ? <div>--</div> : <MONDOLink mondo={wrapper.value}/> },
+      { Header: 'Diagnosis (NCIT)', accessor: 'ncit_id_diagnosis',  Cell: (wrapper) => wrapper.value === "--" ? <div>--</div> : <NCITLink ncit={wrapper.value}/> },
       { Header: 'Diagnosis (Source Text)', accessor: 'source_text_diagnosis', Cell: cellBreak },
       { Header: 'Age at event', accessor: 'age_at_event_days', Cell: cellBreak },
       {
         Header: 'Mondo term shared with',
         accessor: 'shared_with',
         Cell: wrapper => {
 
-          if(wrapper.value === "0 participant") return <div>0 participant</div>;
+          if(wrapper.value === "0 participants") return <div>0 participant</div>;
```

Comment:
@claudia1296 Should the returned div also contain participant with an `s` at the end?